packages/main/src/plugin/index.ts

```diff
@@ -2519,6 +2519,10 @@ export class PluginSystem {
       },
     );
 
+    this.ipcHandle('kubernetes-client:refreshContextState', async (_listener, context: string): Promise<void> => {
```

Comment:
I am bit confused with the wording, the PR title says `restart` and here we have `refresh` is there a semantic reason   ?

Reply:
I would say that one is what is seen from the caller, and the other is what is implemented: the states are refreshed by restarting the informers

Reply:
what the user should see ? a button with restart or refresh ?

Reply:
I think "refresh", as the user doesn't care how it is implemented, by restarting informers, or reinitializing them, etc

Reply:
(I have renamed the PR to clarify)